openssh/ssh_config

```diff
@@ -49,10 +49,10 @@ Port 2222
 #IdentityFile ~/.ssh/id_ssh-picnicL3FS
 #IdentityFile ~/.ssh/id_ssh-sphincsharaka128fsimple
 
-#IdentityFile ~/.ssh/id_ssh-ecdsa-nistp256-dilithium2aes
+IdentityFile ~/.ssh/id_ssh-ecdsa-nistp256-dilithium2aes
 #IdentityFile ~/.ssh/id_ssh-ecdsa-nistp256-falcon512
 #IdentityFile ~/.ssh/id_ssh-ecdsa-nistp256-picnicL1full
-IdentityFile ~/.ssh/id_ssh-ecdsa-nistp384-picnicL3FS
+#IdentityFile ~/.ssh/id_ssh-ecdsa-nistp384-picnicL3FS
```

Comment:
Also, any reference to picnic should be removed to not tempt users into activating a non-supported algorithm again.

Reply:
Thanks for purging picnic from the USAGE file too. To complete things, could you please also kick it from all config files? Would save me a PR just for that...

Reply:
my bad, I misinterpreted what you meant there. will fix now